File: Selenium/Activities/Activity12.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity12 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Home page title is :"+driver.getTitle());
        WebElement textField= driver.findElement(By.id("input-text"));
        System.out.println("Text field enabled status :"+ textField.isEnabled());
        WebElement inputToggle= driver.findElement(By.id("toggleInput"));
        inputToggle.click();
        System.out.println("Text field enabled status :"+ textField.isEnabled());


    }
}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("page title is :", driver.title)
    textbox=driver.find_element(By.ID,"input-text")
    textbox_toggle = driver.find_element(By.ID, "toggleInput")
    print("Text box is enabled: ", textbox.is_enabled())
    # Click the text box toggle button
    textbox_toggle.click()
    # Verify if the text box is enabled or not
    print("Text box is enabled: ", textbox.is_enabled())
    textbox.send_keys("Selenium with Python")